packages/firebase/index.d.ts

```diff
@@ -6060,6 +6061,20 @@ declare namespace firebase.firestore {
      */
     app: firebase.app.App;
 
+    /**
+     * Clears the persistent storage.
+     *
+     * Must be called after the app is shutdown or before other methods (other
+     * than settings()) on startup. If the client is still running, an exception
+     * `code` will be thrown.
+     *
+     *   * failed-precondition: The client is still running in a browser tab.
+     *
+     * @return A promise that is resolved once the persistent storage has been
+     * cleared.
```

Comment:
If IndexedDB fails to clear persistence, does that result in a rejected promise?